livestream-compose-sample/src/main/java/io/getstream/livestream/compose/ui/LiveStreamComment.kt

```diff
@@ -0,0 +1,78 @@
+package io.getstream.livestream.compose.ui
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import io.getstream.chat.android.client.models.name
+import io.getstream.chat.android.compose.state.messages.items.MessageItem
+import io.getstream.chat.android.compose.ui.common.MessageBubble
+import io.getstream.chat.android.compose.ui.common.avatar.UserAvatar
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.livestream.compose.getTimeAgo
+import java.text.SimpleDateFormat
+import java.util.Date
+
+/**
+ * View component to add custom live stream channels screen.
+ *
+ * @param modifier - Modifier for styling.
+ * @param messageItem - Data model for messageItem for each chat message row
+ */
+@Composable
+fun LiveStreamComment(
+    modifier: Modifier = Modifier,
+    messageItem: MessageItem
+) {
+    val context = LocalContext.current
+    Row(
+        modifier = modifier
+            .padding(start = 8.dp, bottom = 2.dp)
```

Comment:
Should this be a spacer or maybe on children components instead of the root?

Reply:
why ? can u share more details , with compose i dont get why not padding here ?

Reply:
We discussed this - if we add padding to the root component we're making that component smaller (and its touch area).

We may not want to that for all examples but it seems like it's okay here. I'm also curious why we have a `start` padding but not `end`.